cephbackupstorage/cephbackupstorage/cephagent.py

```
@@ -1593,12 +1593,20 @@ def run_upgrade_script():
+                    '{0} "sudo -S < {1} chmod +x {2}'
+                    ' && cd {4}'
+                    ' && sudo -S < {1} timeout {3} {2}"'
+                ).format(
                     sshpass_cmd_header,
                     quoted_remote_sudo_passwd_file_path,
                     upgrade_script_path,
-                    script_timeout)
-                r, _, e = bash.bash_roe(cmd_str)
+                    script_timeout,
+                    upgrade_script_dir)
+                r, o, e = bash.bash_roe(cmd_str)
```

Ruff (0.15.9)

[warning] Unpacked variable `o` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)